govc/README.md

```diff
@@ -1,39 +1,49 @@
 # govc
 
-govc is a vSphere CLI built on top of govmomi.
+`govc` is a vSphere CLI built on top of govmomi.
 
 The CLI is designed to be a user friendly CLI alternative to the GUI and well suited for automation tasks.
 It also acts as a [test harness](test) for the govmomi APIs and provides working examples of how to use the APIs.
 
 ## Installation
 
-You can find prebuilt govc binaries on the [releases page](https://github.com/vmware/govmomi/releases).
+### Docker
 
-Download and install a binary locally like this:
+The official `govc` [Docker images](https://hub.docker.com/r/vmware/govc) are built from this [Dockerfile](../Dockerfile.govc).
+
+### Binaries
+You can find prebuilt `govc` binaries on the [releases page](https://github.com/vmware/govmomi/releases).
+
+You can download and install a binary locally like this:
 
 ``` console
-% curl -L $URL_TO_BINARY | gunzip > /usr/local/bin/govc
-% chmod +x /usr/local/bin/govc
+# get version from https://github.com/vmware/govmomi/releases/latest
+$ VERSION=v0.25.0
+$ URL=https://github.com/vmware/govmomi/releases/download/${VERSION}/govc_$(uname -s)_$(uname -m).tar.gz
+
+$ curl -L -o - $URL | tar -Oxvzf - govc > /usr/local/bin/govc
+$ chmod +x /usr/local/bin/govc
```

Comment:
We could skip chmod now that we're using a tar (file permissions preserved), unlike the previous releases that were just single file gzip or zip. So it could be a one-liner:
`curl -L -o - $URL | tar -xvzf- -C /usr/local/bin govc`

Reply:
in my tests, the binary was not executable after extraction :/

Reply:
Did you try the command above?  Or with this you can see the file perms:
```console
$ % curl -s -L -o - $URL | tar -tzvf- 
-rw-r--r--  0 runner docker  10518 Apr 16 09:00 CHANGELOG.md
-rw-r--r--  0 runner docker  11358 Apr 16 09:00 LICENSE.txt
-rw-r--r--  0 runner docker   4437 Apr 16 09:00 README.md
-rw-r--r--  0 runner docker   1812 Apr 16 09:00 license/assignment_manager.go
-rw-r--r--  0 runner docker   4269 Apr 16 09:00 license/manager.go
-rwxr-xr-x  0 runner docker 36778272 Apr 16 09:04 govc
```

Reply:
Thx! Resolved in #2388